next.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
       process.env.HUMAN_INTERNET_URL || "https://human-internet.org/",
     DYNAMIC_LINK_DOMAIN:
       process.env.DYNAMIC_LINK_DOMAIN || "https://devme.bettersocial.org",
+    WIX_URL: process.env.WIX_URL || 'https://home.helio.social/'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a new environment variable `WIX_URL` is fine, but it's important to ensure that this environment variable is properly set in all environments where the application runs (development, staging, production, etc.). If not set, it will default to 'https://home.helio.social/', which might not be the desired behavior in all cases. Please make sure to update your environment configuration files accordingly.
```diff
-    WIX_URL: process.env.WIX_URL || 'https://home.helio.social/'
+    WIX_URL: process.env.WIX_URL || 'https://home.helio.social/' // Ensure this env variable is set in all environments
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->